js/apps/admin-ui/src/PageHeader.tsx

```diff
@@ -104,11 +103,21 @@ const KebabDropdown = () => {
     <Dropdown
       id="user-dropdown-kebab"
       isPlain
-      position="right"
-      toggle={<KebabToggle onToggle={(_event, val) => setDropdownOpen(val)} />}
+      onOpenChange={(isOpen: boolean) => setDropdownOpen(isOpen)}
```

Comment:
This type should already be part of the type definition of `onOpenChange`, right?
```suggestion
      onOpenChange={(isOpen) => setDropdownOpen(isOpen)}
```